### ISSUE TITLE
Very high training accuracy but a very low validation accuracy

### ISSUE THREAD

**Issue by akshatkaushik25** [duggalrahul/AlexNet-Experiments-Keras#4]
Hi,

I am getting a very high training accuracy and it seems to be improving per epoch but a fairly low validation accuracy on a different dataset that I am using on pretrained alexnet code supplied. I have 1598 training observations and 140 validation observations split into 2 classes with equal size. 

Below are the results from some epochs:

> Epoch 1/1
1598/1598 [==============================] - 114s - loss: 0.0889 - acc: 0.8755 - val_loss: 0.2725 - val_acc: 0.5857
Epoch 1/1
1598/1598 [==============================] - 110s - loss: 0.0848 - acc: 0.8874 - val_loss: 0.2937 - val_acc: 0.5286
Epoch 1/1
1598/1598 [==============================] - 114s - loss: 0.0859 - acc: 0.8867 - val_loss: 0.2760 - val_acc: 0.6143
Epoch 1/1
1598/1598 [==============================] - 122s - loss: 0.0787 - acc: 0.8949 - val_loss: 0.2841 - val_acc: 0.5571
Epoch 1/1
1598/1598 [==============================] - 115s - loss: 0.0825 - acc: 0.8880 - val_loss: 0.2466 - val_acc: 0.5786
Epoch 1/1
1598/1598 [==============================] - 113s - loss: 0.0765 - acc: 0.9024 - val_loss: 0.2792 - val_acc: 0.6429
Epoch 1/1
1598/1598 [==============================] - 104s - loss: 0.0774 - acc: 0.9011 - val_loss: 0.2972 - val_acc: 0.5500

> 
Any idea what could be the issue? Is the model overfitting the training set? I could see drop outs being implemented but still why is it overfitting? How can I improve the validation accuracy?

**Comment by duggalrahul** [comment]
Hi Akshat,

You're right about your model overfitting on the training set.

You can experiment with the number of epochs you spend on fine-tuning each CNN layer. Currently the code fine-tunes every layer for 10 epochs. You can possibly increase the number of these, depending upon how close your target dataset is to the ImageNet dataset.

Another thing you can try is increasing the size of your validation set. 140 images seems like a very small number of images.

Additionally, maybe you can try reducing your batch size to accommodate the small validation dataset.

In case you find a solution to your problem, please do share your experience here.

Best,
Rahul

**Comment by alyato** [comment]
@akshatkaushik25 
Do you solve this issue?
how to do it?
thanks.

**Comment by isahhin** [comment]
I have also encountered the same issue.
If anyone solved this problem, please write here.

Thanks.

**Comment by cicinoworld** [comment]
Increase your epoch might help.
